go.mod

```diff
@@ -323,6 +323,8 @@ replace (
 	github.com/go-ldap/ldap/v3 => github.com/YangKeao/ldap/v3 v3.4.5-0.20230421065457-369a3bab1117
 	github.com/pingcap/tidb/pkg/parser => ./pkg/parser
 
+	github.com/tikv/client-go/v2 => ../client-go
```

Comment:
I think this should be removed. I assume this is to do testing together with https://github.com/tikv/client-go/pull/1489 before it is merged?

Reply:
Yes, you're right. I shouldn't have included this local change.

Reply:
Is it the right process?
1. get https://github.com/tikv/client-go/pull/1489 merged
2. update client-go in go.mod to include the change

Reply:
Yes